UIDD/UIDD-Session1/UIDD-Session1.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>UI Design and Development Session 1</title>
+</head>
+<body>
+
+
+	<!--Header-->
+	<header>
+		<div id="headerId" class="header">
+			<img src="images/footer-logo.png">
+			<div id="headerIdOne" class="headerOne">
+				<a href="home.html">Home</a>
```

Comment:
We should not add the links in page which are not available in project. Instead of that use #.